### PR TITLE
Add commit slow log

### PR DESCRIPTION
**Pull request by Connor1996**
### What is changed and how it works?

Issue Number: Close #xxx

What's Changed:

```commit-message
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

```release-note
Please add a release note.

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with None.
If this PR will be picked to release branch, then a release note is probably required.
```

### CODE REVIEW

**Review by zhangjinpeng1987 — APPROVED** [review]
LGTM

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -13215,6 +13215,333 @@
             "alignLevel": null
           }
         },
+        {
```
[the remark this answers is not in the document]
Comment (Connor1996) [reply metrics/grafana/tikv_details.json:13218]: generated by grafana

File: components/raftstore/src/store/peer.rs [hunk 2]
```diff
@@ -1595,6 +1600,26 @@ where
...
+                while let Some((propose_idx, instant)) = self.pending_propose_instants.front() {
+                    if index == *propose_idx {
+                        ctx.raft_metrics
+                            .proposal_send_wait
+                            .observe(instant.elapsed().as_secs_f64());
+                    }
+                    if index >= *propose_idx {
+                        self.pending_propose_instants.pop_front();
+                    } else {
+                        break;
```
[the remark this answers is not in the document]
Comment (Connor1996) [reply components/raftstore/src/store/peer.rs:1618]: you can comment on the original PR https://github.com/tikv/tikv/pull/12837, this temp PR is just to investigate an issue.

**Pull request closed without merging**